views/templates/front/ordersPage.tpl

```diff
@@ -36,7 +36,9 @@
             {foreach from=$orders item=order}
                 <tr>
                     <td>
-                        <b>{$order->getName()|escape:'htmlall'}</b> ({($order->isPaid() == true) ? 'paid' : 'unpaid'}, {($order->isDelivered() == true) ? 'delivered' : 'undelivered'})<br>
+                        <b>{$order->getName()|escape:'htmlall'}</b> 
```

Comment:
You should assign some values directly within the controller, views should not be used to treat anything.